e2e_tests/tests/requirements.txt

```diff
@@ -6,3 +6,5 @@ torchvision==0.5.0
 tensorflow==1.14
 numpy
 pandas
+pyyaml
```

Comment:
question: What are these two packages about? Didn't see any changes related to that.

Reply:
So these were missing dependencies, they didn't show up before because they were being installed as part of `det-deploy`, but for GKE we don't need to include `det-deploy`.